zerver/views/scheduled_messages.py

```diff
@@ -36,14 +37,82 @@ def delete_scheduled_messages(
 
 
 @has_request_variables
-def scheduled_messages_backend(
+def update_scheduled_message_backend(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    scheduled_message_id: int = REQ(converter=to_non_negative_int, path_only=True),
```

Comment:
This `converter=to_non_negative_int` is what is used in the `message_edit` view, so went ahead and updated that here as well since this is now in the path. I wasn't sure if we should also make this change for the delete scheduled message path.

Reply:
That's probably a good idea in that context as well, yeah.

Reply:
#25855 is up with this change.